### PR TITLE
Add sort by relevancy to search page

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5492

image

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -97,7 +97,9 @@ const FilterLabel = styled.label`
 const constructSortByQuery = sortByValue => {
   let query = {};
   if (sortByValue === 'ACTIVITY') {
-    query = { field: 'ACTIVITY', direction: 'DESC' };
+    query = { field: 'ACTIVITY', direction: 'ASC' };
```
Comment (znarf) [inline pages/search.js:100]: Looks like a terrible regression.

**Review by znarf — CHANGES_REQUESTED** [review]
The intent looks good but there is a significant mistake.

File: pages/search.js (hunk 1 above)
Comment (SudharakaP) [reply pages/search.js]: Ops, sorry about that. 🤯 Somehow I missed this one.

**Comment by SudharakaP** [comment]
> The intent looks good but there is a significant mistake.

Thanks. I've corrected the activity ordering mistake.

**Review by znarf — APPROVED** [review]
Ship it.

**Pull request merged**